The Laundry/The_Laundry.html

```diff
@@ -127,7 +127,7 @@ <h3 style="text-align: center; color: #FFF; background-color: #000;"><span data-
                             </td>
                             <td style="text-align: right"><span data-i18n="idea">Idea</span></td>
                             <td>
-                                <input type="number" name="attr_idea" value="@{int} * 5" readonly="readonly" />%
+                                <input type="number" name="attr_idea" value="@{int} * 5" disabled" />%
```

Comment:
The "disabled" key word has an extra " after it. I don't think this will actually break anything, but it should be fixed just in case. This is also the case for several changes below this.